.travis.yml

```diff
@@ -272,43 +272,43 @@ matrix:
-    before_script:
-      - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
-      - cd kitchen-tests
-    script:
-      - bundle exec kitchen test awesome-customers-ubuntu-ubuntu-1404
-    after_failure:
-      - cat .kitchen/logs/kitchen.log
-    env:
-      - AWESOME_CUSTOMERS_UBUNTU=1
-      - KITCHEN_YAML=.kitchen.travis.yml
-  - rvm: 2.3.3
-    services: docker
-    sudo: required
-    gemfile: kitchen-tests/Gemfile
-    before_install:
-      - gem update --system $(grep rubygems omnibus_overrides.rb | cut -d'"' -f2)
-      - gem install bundler -v $(grep bundler omnibus_overrides.rb | cut -d'"' -f2)
-    bundler_args: --without ci development docgen guard integration maintenance omnibus_package tools aix bsd mac_os_x solaris windows --frozen
-    before_script:
-      - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
-      - cd kitchen-tests
-    script:
-      - bundle exec kitchen test awesome-customers-rhel-centos-7
-    after_failure:
-      - cat .kitchen/logs/kitchen.log
-    env:
-      - AWESOME_CUSTOMERS_RHEL=1
-      - KITCHEN_YAML=.kitchen.travis.yml
-    ### END TEST KITCHEN ONLY ###
+#  - rvm: 2.3.3
+#    services: docker
+#    sudo: required
+#    gemfile: kitchen-tests/Gemfile
+#    before_install:
+#      - gem update --system $(grep rubygems omnibus_overrides.rb | cut -d'"' -f2)
+#      - gem install bundler -v $(grep bundler omnibus_overrides.rb | cut -d'"' -f2)
+#    bundler_args: --without ci development docgen guard integration maintenance omnibus_package tools aix bsd mac_os_x solaris windows --frozen
+#    before_script:
+#      - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
+#      - cd kitchen-tests
+#    script:
+#      - bundle exec kitchen test awesome-customers-ubuntu-ubuntu-1404
+#    after_failure:
+#      - cat .kitchen/logs/kitchen.log
+#    env:
+#      - AWESOME_CUSTOMERS_UBUNTU=1
+#      - KITCHEN_YAML=.kitchen.travis.yml
+#  - rvm: 2.3.3
+#    services: docker
+#    sudo: required
+#    gemfile: kitchen-tests/Gemfile
+#    before_install:
+#      - gem update --system $(grep rubygems omnibus_overrides.rb | cut -d'"' -f2)
+#      - gem install bundler -v $(grep bundler omnibus_overrides.rb | cut -d'"' -f2)
+#    bundler_args: --without ci development docgen guard integration maintenance omnibus_package tools aix bsd mac_os_x solaris windows --frozen
+#    before_script:
+#      - sudo iptables -L DOCKER || ( echo "DOCKER iptables chain missing" ; sudo iptables -N DOCKER )
+#      - cd kitchen-tests
+#    script:
+#      - bundle exec kitchen test awesome-customers-rhel-centos-7
+#    after_failure:
+#      - cat .kitchen/logs/kitchen.log
+#    env:
+#      - AWESOME_CUSTOMERS_RHEL=1
+#      - KITCHEN_YAML=.kitchen.travis.yml
+#    ### END TEST KITCHEN ONLY ###
```

Comment:
no, that's going to get uncommented by the time that chef-13 goes live.  its blocked on chef_gem_mysql2 and mysql 8.x cookbooks being used by those tests, but learnchef needs to make changes to get onto mysql 8.x and the changes to chef_gem_myxql2 that it needs are committed to master but not yet released.  the mysql 8.x issue is also complicated by the fact that adding the repo has been removed from the mysql cookbook so learnechef needs to pick that up explicitly or something like that.  so there's a whole in-progress yak shave going on here...